templates/action.epp

```diff
@@ -25,9 +25,13 @@ action(type="<%= $type %>"
     name="<%= $action_name %>"
 <%}-%>
 <% if $config { -%>
```

Comment:
if you like you can also remove this if check. It doesn't make any sense anymore. We can always iterate on $config. If it's empty it won't print anything.